lib/backend/backend.go

```diff
@@ -305,10 +305,10 @@ func (p Params) GetString(key string) string {
 // NoLimit specifies no limits
 const NoLimit = 0
 
-// nextKey returns the next possible key.
+// NextKey returns the next possible key.
 // If used with a key prefix, this will return
 // the end of the range for that key prefix.
-func nextKey(key []byte) []byte {
+func NextKey(key []byte) []byte {
```

Comment:
Undo this please, it's far too easy to misuse (and it's already exported as `RangeEnd` anyway).